pkg/config/config.go

```diff
@@ -312,6 +313,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("jmx_thread_pool_size", 3)
 	config.BindEnvAndSetDefault("jmx_reconnection_thread_pool_size", 3)
 	config.BindEnvAndSetDefault("jmx_collection_timeout", 60)
+	config.BindEnvAndSetDefault("jmx_check_period", defaults.DefaultCheckInterval/time.Millisecond)
```

Comment:
Document the setting in `config_template.yaml`?

Reply:
Good catch, I merged too fast! Going to address it in a new PR

Reply:
My bad, addressed here: https://github.com/DataDog/datadog-agent/pull/3478